### PR TITLE
Removed table styles: Issue: #329

### PR DESCRIPTION
**Pull request by shawnthompson**

### CODE REVIEW

File: src/_defaults.scss [hunk 1]
```diff
@@ -104,30 +104,6 @@ h6,
...
-		&:hover,
-		&:focus {
-			color: #000;
-			text-decoration: none;
-		}
-	}
-}
-
 td {
 	border: 1px solid #ddd;
```
Comment (LaurentGoderre) [inline src/_defaults.scss]: why are you not removing those?

**Comment by shawnthompson** [comment]
I'll fix that... not sure what happened. Went too fast probably

**Comment by shawnthompson** [comment]
That's not the code I removed locally, Although I did forget to remove the td rule.
Amending now and going to try a forced push. Hopefully this cleans it up

**Comment by shawnthompson** [comment]
@LaurentGoderre How's it look now?

**Comment by LaurentGoderre** [comment]
Probably best to remove the caption style as well

**Comment by shawnthompson** [comment]
Done!

**Comment by LaurentGoderre** [comment]
LGTM

**Pull request merged**